.taprc

```diff
@@ -0,0 +1,6 @@
+test-env:
+  - DB_USER=postgres
+  - DB_PORT=5432
+  - DB_NAME=postgres
+  - DB_PASSWORD=postgres
+  - DATABASE_URL=postgres://postgres:postgres@0.0.0.0:5432/postgres?sslmode=disable
```

Comment:
I know I suggested using this earlier, but now that we set these vars in our github workflow I don't think we need it. Could we see if tests pass without it?